warehouse/admin/views/projects.py

```diff
@@ -465,3 +466,20 @@ def delete_project(project, request):
     remove_project(project, request)
 
     return HTTPSeeOther(request.route_path("admin.project.list"))
+
+
+@view_config(
+    route_name="admin.project.reindex",
+    permission="admin",
```

Comment:
I think we can make this available to moderators, it's not destructive.

Reply:
Cool. Just pushed that change. View is now allowing moderators and I removed the superuser checks from the DOM.